src/components/Coupon/index.test.jsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import renderer from "react-test-renderer";
+import Coupon from "./index";
+
+jest.mock("react-i18next", () => ({
+  useTranslation: () => ({ t: (key) => key }),
+}));
+
+const isTesting = true;
```

Comment:
no need for decleration, just pass isTesting to the component like: `<Component isTesting />`